src/openrct2/ride/Ride.h

```diff
@@ -27,6 +27,7 @@ class StationObject;
 struct Peep;
 struct Ride;
 struct Staff;
+struct RideTypeDescriptor;
```

Comment:
```suggestion
struct RideTypeDescriptor;
struct Staff;
```
Just to keep sorted.